source/tests/system/nidcpower_session_tests.cpp

```diff
@@ -0,0 +1,149 @@
+#include <gtest/gtest.h>
+
+#include "nidcpower/nidcpower_library.h"
+#include "nidcpower/nidcpower_service.h"
+
+namespace ni {
+namespace tests {
+namespace system {
+
+namespace dcpower = nidcpower_grpc;
+
+const int kViErrorRsrcNFound = -1073807343;
+const char* kViErrorRsrcNFoundMessage = "VISA:  (Hex 0xBFFF0011) Insufficient location information or the device or resource is not present in the system.";
+const char* kTestResourceName = "FakeDevice";
+const char* kSimulatedOptionsString = "Simulate=1, DriverSetup=Model:4147; BoardType:PXIe";
+const char* kTestSessionName = "SessionName";
+const char* kInvalidResourceName = "";
+
+class NiDCPowerSessionTest : public ::testing::Test {
```

Comment:
@ni-pkhare , you need to make changes in cmakelists.txt to add this file to systemtestsrunner

Reply:
Yes, made the necessory changes.